### ISSUE TITLE
Add a filter for "sources" only

### ISSUE THREAD

**Issue by styfle** [Hacksore/gitpilled#3]
It looks like all forks are included in the stats since [JS is number 1 for me](
https://github.com/Hacksore/gitpilled/assets/229881/83752838-1d4e-4fca-89dd-0883b17ac4c0).

https://github.com/styfle?tab=repositories

However, if I exclude "forks" and only look at "sources", then I think TS would be number 1.

https://github.com/styfle?tab=repositories&type=source

**Comment by Hacksore** [comment]
yeah @typesafeui has posted this on twitter but tl;dr the hub API sucks.

https://x.com/typesafeui/status/1790108971262415029

**Comment by sabahmax-dev** [comment]
Hey! I’ve opened PR #4 to address this — it adds a “sources only” filter so the stats can exclude forks and show only source repositories. 🚀
